coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentDocumentRule.java

```diff
@@ -653,7 +653,13 @@ public boolean processProposalDataOverrideRules(ProposalDataOverrideEvent propos
     }
     
     public boolean processBudgetDataOverrideRules(BudgetDataOverrideEvent budgetDataOverrideEvent) {
+        try {
         return new ProposalBudgetDataOverrideRule().processBudgetDataOverrideRules(budgetDataOverrideEvent);
+        } catch (NullPointerException e) {
+          LOG.warn("NPE caught when calling: new ProposalBudgetDataOverrideRule().processBudgetDataOverrideRules(budgetDataOverrideEvent)");
+          LOG.warn("returning false");
+          return false;
+        }
     }
     
```

Comment:
Like I said, I am commenting on stuff that really needs cleaning up. We do not have to do it now but will need to at some point. This one seems less harmful than the other instance I noticed.